analyses/org.osate.analysis.resource.budgets/plugin.xml

```diff
@@ -265,13 +265,20 @@ censes only apply to the Third Party Software and not any other portion of this
          </command>
       </menuContribution>
    </extension>
-   <extension
-         id="toc_busload"
-         name="Bus Load Analysis Help Table of Contents"
-         point="org.eclipse.help.toc">
-      <toc
-            file="help/toc_busload.xml"
-            primary="true">
-      </toc>
```

Comment:
This doesn't seem right.

Reply:
Good catch.  Not sure what happened here.

Reply:
fixed it

Reply:
Help text is still missing. Could you do a local build and test that the help is _really_ there?